packages/tabs/tab.md

```diff
@@ -7,93 +7,84 @@ An `<sp-tab>` element surfaces a `label` attribute to serve as its default text
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab vertical value="4">
-        Tab 4
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab-panel value="1">
-        Content for Tab 1 which uses an attribute for its content delivery
-    </sp-tab-panel>
-    <sp-tab-panel value="2">
-        Content for Tab 2 which uses its text content directly
-    </sp-tab-panel>
-    <sp-tab-panel value="3">
-        Content for Tab 3 which uses an attribute with a
-        <code>[slot="icon"]</code>
-        child
-    </sp-tab-panel>
-    <sp-tab-panel value="4">
-        Content for Tab 4 which uses both its text content and a
-        <code>[slot="icon"]</code>
-        child displayed using the
-        <code>[vertical]</code>
-        attribute to define their alignment
-    </sp-tab-panel>
+<sp-tab value="complete" aria-label="Tab with checkmark">
+    <sp-icon-checkmark
+        slot="icon"
+        label="Checking your work"
+    ></sp-icon-checkmark>
+</sp-tab>
+```
+
+#### Label only
+
+```html
+<sp-tab value="label" label="Label"></sp-tab>
+```
+
+The `label` attribute provides the visible text content and accessible name for screen readers. The `value` attribute links the tab to its corresponding `sp-tab-panel` via matching values.
+
+### States
+
+In order to activate the `<sp-tab>` element's interactive states, it must be used within an `<sp-tabs>` element.
+
+#### Selected state
+
+The tab is currently active and its associated panel is visible.
+
+```html
+<sp-tabs>
+    <sp-tab selected="label" label="Label" value="1"></sp-tab>
 </sp-tabs>
 ```
 
+#### Focused state
+
+The tab has keyboard focus. All tabs can receive focus through keyboard navigation, except when the tab is `disabled`.
+
 ```html
-<sp-tabs selected="1" direction="vertical">
+<sp-tabs selected="2">
     <sp-tab label="Tab 1" value="1"></sp-tab>
-    <sp-tab value="2">Tab 2</sp-tab>
-    <sp-tab label="Tab 3" value="3">
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab vertical value="4">
-        Tab 4
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab-panel value="1">
-        Content for Tab 1 which uses an attribute for its content delivery
-    </sp-tab-panel>
-    <sp-tab-panel value="2">
-        Content for Tab 2 which uses its text content directly
-    </sp-tab-panel>
-    <sp-tab-panel value="3">
-        Content for Tab 3 which uses an attribute with a
-        <code>[slot="icon"]</code>
-        child
-    </sp-tab-panel>
-    <sp-tab-panel value="4">
-        Content for Tab 4 which uses both its text content and a
-        <code>[slot="icon"]</code>
-        child displayed using the
-        <code>[vertical]</code>
-        attribute to define their alignment
-    </sp-tab-panel>
+    <sp-tab label="Tab 2" value="2"></sp-tab>
+    <sp-tab label="Tab 3" value="3"></sp-tab>
 </sp-tabs>
 ```
```

Comment:
Since we can't show focused without stealing keyboard focus, we could probably remove the example here.